gulpfile.js

```diff
@@ -27,7 +31,7 @@ require('gulp-task-loader')({
 // Define task for each theme, locale, lang, processing type etc.
 // Gulp can't run same task in parallel, so we need different names
 Object.keys(config.themes).forEach(name => {
-  var theme = config.themes[name];
+  const theme = config.themes[name];
   theme.locale.forEach(locale => {
     gulp.task(
       theme.lang + ':' + name + ':' + locale,
```

Comment:
I think if we have a task to setup configuration, throwing errors while `themes.json` doesn't exist might not be so useful anymore.
To be honest I don't like idea of `get-themes`. IMO it's pointless to keep this as separate helper.

My goal is to keep things simple, but in this case simplifying tools for noobs (sorry, but it's super easy to copy / create configuration in right place) are adding complexity to whole project.